doc/tietomalli.md

```diff
@@ -1,56 +1,103 @@
 
-Tässä kuvassa esitetään geoviitteen peruskäsitteet ja miten ne liittyvät toisiinsa. Käsitteistöä on tarkoituksella
-yksinkertaistettu ollakseen havainnollisempi ja luokkahierarkiat eivät vastaa 1-1 tietokantatauluja vaan ennenkaikkea
+Tässä dokumentissa esitetään Geoviitteen peruskäsitteet ja miten ne liittyvät toisiinsa. Käsitteistöä on tarkoituksella
+yksinkertaistettu ollakseen havainnollisempi ja luokkahierarkiat eivät vastaa 1-1 tietokantatauluja vaan ennen kaikkea
 käsitteistön kannalta oleellista osaa niiden tietosisällöstä.
 
 ![](images/data_flow.png)
 
+## Käsitteiden 3 esitystapaa
+
+| Käsite          | Geometria                                                                                                                                                                                                                            | Paikannuspohja                                                                                                                                                                                                                      | Osoitepisteet                                                                                                                                                                                                                                                                                                              |
+| --------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Yleiskuva       | Tarkat suunnitelluohjelmissa tuotetut matemaattisesti kuvatut geometriat. Toisistaan riippumattomia, eri koordinaatistoissa olevia lähtötietoja, jotka ovat osin päällekkäisiä ja eivät kuvaa kaikkia alueita lainkaan.              | Koko rataverkko, tuotuna yhtenäiseen koordiinaatistoon, helposti kartalla esitettävässä muodossa.                                                                                                                                   | RATKO:n esitysmuoto. Koostuu tasametripisteistä jotka on valittu projisoimalla pituusmittauslinjalta, ja johon on siten sisällytetty rataosoitejärjestelmä. Tasametrien lisäksi sisältää myös epätasamatripisteinä tärkeät raiteella olevat kohdat, kuten alku- ja loppupisteen sekä mahdolliset vaihteiden jatkospisteet. |
```

Comment:
"suunnitelluohjelmissa" -> "suunnitteluohjelmissa". Lisäksi en ole ihan varma viitataanko tuolla "... ja eivät kuvaa kaikkia alueita lainkaan" siihen, että kukin suunnitelma kattaa vain pienen osan rataverkosta eikä sitä kokonaan (näin itse käsitin tuon aluksi) vai sitä, että ihan jokaiselle rataverkon kohdalle ei ole suunnitelmia, mutta kirjoittaisin tuon itse ehkä jotenkin tähän tapaan: "Tarkat suunnitteluohjelmissa tuotetut, matemaattisesti kuvatut geometriat. Kukin suunnitelma on toisista suunnitelmista riippumaton ja kuvaa pienen osan rataverkkoa. Suunnitelmissa voi olla päällekkäisyyttä ja jokaiselle rataverkon kohdalle ei löydy suunnitelmia"

Reply:
Ratko-solussa: "epätasamatripisteinä" -> "epätasametripisteinä"